tests/integration/it_utils.py

```diff
@@ -124,6 +143,22 @@ async def sign_and_reliable_submission_async(
     return response
 
 
+def send_timed_reliable_submission(
```

Comment:
Can you add docstrings explaining what this function does + what the args are for?